src/components/ScrollView.md

```diff
@@ -28,7 +28,7 @@ let make = () => {
   <ScrollView ref=scrollViewRef>
     <TouchableOpacity
       onPress={_ =>
-        switch (scrollViewRef->React.Ref.current->Js.Nullable.toOption) {
+        switch (scrollViewRef.React.current->Js.Nullable.toOption) {
```

Comment:
Why?

Reply:
Because it's a record now, it's annotating the record field with the React module.

Reply:
I just found it weird to have a module that is "inside" scrollViewRef 🤷‍♂️

Reply:
It's not a module, it's just a way to bring the module in scope for that particular field (local open does the same). But it might not be necessary (Usually type inference works). This is what the codemod produces.

Reply:
Yeah I get it now. I didn't know we could write that in reason!